plugins/pulp_docker/plugins/models.py

```diff
@@ -284,6 +283,32 @@ def from_json(cls, manifest_json, digest):
                    amd64_digest=amd64_digest,
                    amd64_schema_version=amd64_schema_version)
 
+    @staticmethod
+    def validate_json(manifest_json):
```

Comment:
please rename this so it clear the method validates just json of a manifest list and is not capable to validate json of an image manifest.

Reply:
would be a plus to rename variables cordially ^^

Reply:
(1) Since this is a function on `class ManifestList`, I think it is clear that it intended only for Manifest Lists and not an Image Manifest. 

(2) I chose `manifest_json` to be consistent with the `from_json` function.